IBM/ibm-common-service-operator
# Remove wildcard permisisons

See commit history for details and explanation of changes

## How to test with temporary bundle

1. Checkout this PR `git fetch upstream pull/<pull number>/head:pr-branch`
1. Export `REGISTRY=quay.io/username` and `QUAY_REGISTRY="$REGISTRY"`
1. Run `make build-dev-image`
1. Edit `manager.yaml` to change operator image and the pull policy to Always
1. Run `make bundle-manifests`
1. Run `make build-bundle-image`
1. Run `make run-bundle`
1. The operator should function normally
